### ISSUE TITLE
Created ok for the module dependencies but data folder not seen?

### ISSUE THREAD

**Issue by gweeyc** [nus-tic4001-AY2122S1/forum#15]
Hi Profs and classmates,

![image](https://user-images.githubusercontent.com/43993134/133591127-4d34af2f-6d73-463f-b246-e97cb33205c6.png)

I have the modules set up right as per the instructions in Adding JUnit support to your project and all seems well:

![image](https://user-images.githubusercontent.com/43993134/133591879-21e4dfc4-05e0-47ca-9317-37f32baab010.png)

But now I cannot see my data folder that holds my tasks.txt and backup.txt? But I run the program, Intellij still works fine so I can write to the tasks.txt file with no problem from the program? How do I make the data folder visible again? Do I need to mark this folder as resources?

Strangely I notice the backup.txt file is visible under main folder, but not tasks.txt? I am confused here.
When I try to clear the content from backup.txt I got this prompt -
![image](https://user-images.githubusercontent.com/43993134/133592616-894eeb12-a179-469d-abf4-1530f18d931b.png)

I do not understand?

Thank you for your kind help in advance!

**Comment by damithc** [comment]
@gweeyc you can see the other files by changing this option
![image](https://user-images.githubusercontent.com/1673303/133593886-2935b94d-59ef-4344-95e0-8afffdf0516c.png)

**Comment by damithc** [comment]
> When I try to clear the content from backup.txt I got this prompt -
> ![image](https://user-images.githubusercontent.com/43993134/133592616-894eeb12-a179-469d-abf4-1530f18d931b.png)

Choose the second option and see if that works out.

**Comment by gweeyc** [comment]
HI Prof Damith,
I found out that file was from an earlier attempt, now no longer relevant, I've deleted it fine.

![image](https://user-images.githubusercontent.com/43993134/133595103-05e06a1e-53fd-4832-bad3-02103ba2674b.png)

Now when I change the Project view to Project Files I see the above. Thank you.

My last query here is: do I leave it at that, or is it better to move the data folder under main/java folder? Or is there any need to mark that data folder as resource folder?

Thanks!
